assets/js/common/OperationsButton/OperationsButton.jsx

```diff
@@ -11,8 +11,34 @@ import {
 import { EOS_MORE_VERT, EOS_LOADING_ANIMATED } from 'eos-icons-react';
 
 import Button from '@common/Button';
+import DisabledGuard from '@common/DisabledGuard';
 
-function OperationsButton({ operations }) {
+// The custom component is required to wrap in the DisabledGuard component
+// and pass the disabled value to the child
+function CustomMenuButton({ value, running, disabled, onClick }) {
```

Comment:
What about moving to an own component  ?

Reply:
I don't find it necessary. It is an internal technical decision and I don't want to export the component